platform/ext/target/nordic_nrf/common/core/startup.h

```diff
@@ -28,6 +28,8 @@ __NO_RETURN void __attribute__((weak, alias("default_tfm_IRQHandler"))) handler_
 
 __NO_RETURN void Reset_Handler(void);
 
+void SPU_IRQHandler(void);
```

Comment:
This build system bug has happened before. For the other types of fault.
Is it a regression since then for SPU possibly?

Reply:
FYI: https://github.com/nrfconnect/sdk-trusted-firmware-m/commit/4e1d40b1e108ce9c0f7fa610bf69f65c0fd28305

Reply:
I would assume it is a regression since then yes :)

I could try adding the source to tfm_s.

But IMHO it is more obvious where the SPU_IRQ comes from the way I did than to have a default weak entry that is dead code.

Reply:
Sure, I don't object to this change, but I think it could be extended to include the rest of the faults as well, so we don't end up with the same bug again.

Reply:
It makes more sense to hav ethe startup file in tfm_s, since that is the lib where startup_nrf5340.c is for example.

Reply:
startup.c is now added to tfm_s